src/vs/workbench/api/common/extHostConfiguration.ts

```diff
@@ -201,12 +201,6 @@ export class ExtHostConfigProvider {
 						case 'enableCellCodeLens':
 							return false;
 					}
-				} else if (section === 'python') {
-					switch (key) {
-						// Force to 'Default' to disable the Jupyter extension's language clients.
```

Comment:
I checked through the latest version of the Jupyter source code and I don't think returning `None` instead of `Default` will affect it at all (though it will positively affect basedpyright).